src/h3/api/_api_template.py

```diff
@@ -1,4 +1,4 @@
-import h3.core as c
+from .. import _internal_api as _a
```

Comment:
I am a little concerned about names like `c` or `_a`. They are not descriptive and I think this name has already been renamed. Is there an abstract name for what this is?

Reply:
I have a whole philosophy around this ;)

I think longer, descriptive names are important when you have a large namespace to hold in your head and the names help you remember what that object is. An alternative strategy is to keep namespaces small so you don't have to keep track of lots of things.

In this file there's only one "thing" being imported from the outside, so I'd say its pretty easy to remember what's going on. Pretty much any name would work, but I think shorter names keep the code easier to read. (I'm even a little upset I have to use two whole characters here, but the underscore helps hide the symbol from the end user.)

The example I always think of is a squaring function like

```python
def square(thing_to_be_squared):
    result_of_squaring = thing_to_be_squared*thing_to_be_squared

    return result_of_squaring
```

when something like

```python
def square(x):
    return x*x
```

is probably **more** clear, even though `x` isn't as descriptive of a name. The namespace of the function is so small that the name doesn't really matter, as everything can be figured out from context.

That said, I'm still completely open if you have other suggestions!

Reply:
I understand that, although I would prefer we avoid renaming this in future since it creates large diffs.

Reply:
Good point. I'm still "getting a feel" for the right way to design this wrapper, so a few large diffs might still happen as I rearrange things. I've been considering that the cutoff point for that kind of refactoring would be when we merge this branch into `master`.